internal/common/test_configuration.go

```diff
@@ -154,6 +154,15 @@ var TestIPv6Networking = TestNetworking{
 	IngressVip:                "1001:db8::65",
 }
 
+var TestDualStackNetworking = TestNetworking{
+	ClusterNetworks:           []*models.ClusterNetwork{{Cidr: "1.3.0.0/16", HostPrefix: 24}, {Cidr: "1003:db8::/53", HostPrefix: 64}},
+	ServiceNetworks:           []*models.ServiceNetwork{{Cidr: "1.2.5.0/24"}, {Cidr: "1002:db8::/119"}},
+	PrimaryMachineNetworkCidr: "1.2.3.0/24",
```

Comment:
I see that you replaced `PrimaryMachineNetworkCidr` with `MachineNetworks` in lots of places.
Do we still need the former definition at all?

Reply:
Good point, I have removed the former